CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added a way to initialize the `SrlBert` model without caching/loading pretrained transformer weights.
   You need to set the `bert_model` parameter to the dictionary form of the corresponding `BertConfig` from HuggingFace.
   See [PR #257](https://github.com/allenai/allennlp-models/pull/257) for more details.
+- Added support for Flickr30k image retrieval, including a data loader, model, and training config.
```

Comment:
Should this say NLVR2 instead of Flickr30k?

Reply:
Yes :) I think I got my wires crossed when I was switching back and forth between Flickr30k and NLVR2.